tests/unit_tests/commandline/test_geoips_get_plugin.py

```diff
@@ -0,0 +1,128 @@
+"""Unit test for GeoIPS CLI `get plugin` command.
+
+See geoips/commandline/ancillary_info/cmd_instructions.yaml for more information.
+"""
+
+from numpy.random import rand
+import pytest
+
+from geoips import interfaces
+from tests.unit_tests.commandline.cli_top_level_tester import BaseCliTest
+
+
+class TestGeoipsGetPlugin(BaseCliTest):
+    """Unit Testing Class for Get Plugin Sub-Command."""
+
+    rand_threshold = 0.10
+
+    @property
+    def all_possible_subcommand_combinations(self):
+        """A stochastic list of commands used for the GeoipsGetPlugin command.
+
+        This includes failing cases as well.
+        """
+        if not hasattr(self, "_cmd_list"):
+            self._cmd_list = []
+            base_args = self._get_plugin_args
+            # validate all plugins from package geoips_clavrx
+            for interface_name in interfaces.__all__:
+                interface = getattr(interfaces, interface_name)
+                interface_registry = interface.plugin_registry.registered_plugins[
+                    interface.interface_type
+                ][interface_name]
```

Comment:
@jsolbrig  @evrose54  @biosafetylvl5   Do we want this to fail if we have an interface that happens to have no plugins installed?  Right now this fails if recenter_tc is not installed, because the "sector_adjusters" interface has no sector adjuster plugins installed in the main geoips repo.

Reply:
No, this should be fixed to allow empty interfaces.

Reply:
https://github.com/NRLMMD-GEOIPS/geoips/pull/494/files

I put some fixes to make it work with only geoips installed there - we can just immediately merge here if those fixes are ok.

Reply:
@evrose54  this was resolved? Can you please include a link to the snippet of code where it was fixed?

Reply:
This was resolved. Now, if `interface_name` isn't found in the list of interfaces in the registry, it is skipped via `continue`.

This is fine for this PR, however, I have asked @evrose54 to update the registry creation to always add dictionaries for every interface, even if no plugins exist for a given interface. This will be more explicit and make it easier to understand what is happening in cases where no plugins exist. He created #537 to address this.